mkdocs.yml

```diff
@@ -22,4 +22,27 @@ markdown_extensions:
     custom_checkbox: true
 - pymdownx.superfences
 - toc:
-    permalink: true
\ No newline at end of file
+    permalink: true
+
+# Table of contents
+nav:
+  - index.md
```

Comment:
Why would you do this manually?

Reply:
This is needed to be able to control the information architecture - the order of the entries on the sidebar table of contents, as well as the names of the "parent folders" aggregating subpages.

This is important because otherwise the pages will appear in alphabetical order on the sidebar, instead of a logical order that makes sense for the users.